.github/workflows/build-deploy.yaml

```diff
@@ -78,7 +76,7 @@ jobs:
         uses: Azure/k8s-deploy@v4
         with:
           manifests: |
-            namespaces/default/site/deployment.yaml
+            infra/kubernetes/namespaces/default/site/deployment.yaml
```

Comment:
Shouldn't this be kubernetes/namespaces/... e.g without the infra/ as top level directory since when you checkout you're cd'd into the repo dir?

Reply:
Yeah, I've missed that..